### DISCUSSION TITLE
Withlist

### DISCUSSION THREAD

**Discussion by daili0015** [openxrlab/xrnerf#11] [Ideas]
### Welcome notes
Please leave a comment on which new method/feature/dataset you would like us to add in XRNeRF.
The most upvoted suggestions will be prioritized!

**Comment by QianyiWu** [comment]
Nice framework! It would be better if xrnerf could also support [RefNeRF](https://dorverbin.github.io/refnerf/)?

**Comment by daili0015** [comment]
Support for [DVGO](https://github.com/sunset1995/DirectVoxGO)

> https://github.com/openxrlab/xrnerf/issues/12

**Comment by l-mengwei** [comment]
Nice! This is a good tool for learningNeRF related content. It would be even better if xrnerf could also support [AD-NeRF] ?

**Comment by liwenssss** [comment]
Is there any possible to support 
[ml-neuman](https://github.com/apple/ml-neuman) ?

**Comment by imbinwang** [comment]
Nice framework. It would be better to support some 3d reconstruction work, such as [Neus](https://github.com/Totoro97/NeuS) or [volsdf](https://github.com/lioryariv/volsdf).

**Comment by jeejeelee** [comment]
Nice framework! It would be better if xrnerf could also support [pixelNeRF](https://arxiv.org/abs/2012.02190)?

**Comment by wrk226** [comment]
Great framework! I wonder if you have any interest in supporting [SCARF](https://github.com/yfeng95/SCARF).

**Comment by yzt-cqu** [comment]
Great framework! I wonder if you have any interest in supporting [pointnerf](https://github.com/yzt-cqu/pointnerf).

**Comment by chufall** [comment]
Great framework! I wonder if you have any interest in supporting  "grid-guided nerf"

**Comment by daocidi666** [comment]
how can i connect with? i got some problem with my installation
